packages/rescript-relay-router/vite-plugins/RescriptRelayVitePlugin__ManifestTransform.mjs

```diff
@@ -1,27 +1,35 @@
 // Generated by ReScript, PLEASE EDIT WITH CARE
 
 import * as Fs from "fs";
-import * as Js_dict from "rescript/lib/es6/js_dict.js";
-import * as Belt_Array from "rescript/lib/es6/belt_Array.js";
-import * as Belt_Option from "rescript/lib/es6/belt_Option.js";
-import * as Caml_option from "rescript/lib/es6/caml_option.js";
+import * as Core__Array from "@rescript/core/src/Core__Array.mjs";
```

Comment:
I wonder if this might cause problems because that file won't exist unless it's built. 

I think we either need to let this manifest file continue using only things shipped with the compiler (which means waiting with Core for that file until v12) or bundle it so it has no deps.

Reply:
Well this file is the only reason we're limited to use .mjs, i think bundling it would remove this limitation too, right?